### PR TITLE
refactor: follow to use message method for following

### PR DESCRIPTION
**Pull request by nsylke**
### Items Changed

- [x] Commands
- [ ] Events
- [ ] Handlers
- [ ] Structures
- [ ] Tasks
- [ ] Documentation
- [ ] Other: ______

### Description

 - Update method to follow channels
 - Remove ability to follow channel in another channel (this fixes the webhook size check)

### Issues

N/A

### Have You...?

- [x] I have read the [Contributing Guidelines](https://github.com/typicalbot/typicalbot/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [pull request template](https://github.com/typicalbot/typicalbot/blob/master/.github/PULL_REQUEST_TEMPLATE.md).
- [x] I have checked open [pull requests](https://github.com/typicalbot/typicalbot/pulls) for upcoming features and fixes.

### CODE REVIEW

**Comment by nsylke** [comment]
Blocked until #327 is merged. Additionally, change pointing branch to 4.x once #327 is merged

**Review by Skillz4Killz — APPROVED** [review]

**Comment by Skillz4Killz** [comment]
Although I see a TSC error failing here, i think it should be fixed one DJS version is updated in package.json so CI installs the latest.

**Pull request closed without merging**
